source/logs.txt

```diff
@@ -57,5 +57,5 @@ history:
 Log Retention
 -------------
 
-``mongosh`` retains log files for 30 days. Log files older than
+``mongosh`` retains up to 100 log files for 30 days. Log files older than
 30 days are automatically deleted.
```

Comment:
[suggestion] I know this wasn't part of your ticket changes but it might be worth changing the last sentence to `[active voice](https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/)`!

```suggestion
``mongosh`` retains up to 100 log files for 30 days. MongoDB automatically
deletes log files older than 30 days.
```

Reply:
Let's use "mongosh" or "The MongoDB Shell" instead of "MongoDB" in that case? :)